CHANGES.txt

```diff
@@ -160,6 +160,8 @@ Coming in build 307, as yet unreleased
 * Remove outdated and unused remote feature (#2098, @Avasam)
 * Migrated from `distutils` to `setuptools` (#2133, @Avasam)
 
+* Add EnumDesktopWindows (#2219, @CristiFati)
```

Comment:
I don't think you meant to add this under adodbapi

Reply:
Thanks, it was automatically done by rebase.